engine-rest/engine-rest-openapi/src/main/templates/paths/history/activity-instance/get.ftl

```diff
@@ -0,0 +1,68 @@
+<#assign requestMethod="GET"/>
+<#include "/lib/commons/history-activity-instance.ftl" >
```

Comment:
Is it positioned here due to the assigning of the sorting values? It might be unnoticed at the beginning of the file.
Could you consider do the order of the includes in the parameters section as follows:
```
    <#assign last = false >

    <#assign requestMethod="GET"/>
    <#include "/lib/commons/history-activity-instance.ftl" >

    <#include "/lib/commons/pagination-params.ftl" >
    <#include "/lib/commons/sort-params.ftl" >

    <@lib.parameters
        object = params
        last = true/>
```

Reply:
@yanavasileva, we can do that! However, I would prefer to put the `<#assign last = false >` close to `<#include "/lib/commons/pagination-params.ftl" >`. What do you think about the following:

```
    <#assign requestMethod="GET"/>
    <#include "/lib/commons/history-activity-instance.ftl" >

    <#assign last = false >
    <#include "/lib/commons/sort-params.ftl" >

    <#include "/lib/commons/pagination-params.ftl" >

    <@lib.parameters
        object = params
        last = true/>
```

Reply:
@tasso94 👍  looks good as well

Reply:
👍